com.unity.probuilder/Runtime/MeshOperations/MeshValidation.cs

```diff
@@ -0,0 +1,306 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace UnityEngine.ProBuilder.MeshOperations
+{
+    /// <summary>
+    /// Methods for testing that mesh topology is correct.
```

Comment:
This class seems to do more than just testing ProBuilderMeshes with the Rebuild/Remove methods. Maybe add a few words more info about.

Reply:
14789d9c